src/snorkel_extractor_location/Snorkel_Pipeline_Version_NB3.ipynb

```diff
@@ -842,7 +284,7 @@
     "geocode_key = None\n",
     "# geocode_key = 'AIzaSyBlLyOaasYMgMxFGUh2jJyxIG0_pZFF_jM'\n",
     "\n",
-    "doc_extractions = create_extractions_dict(session, eval_cands, eval_marginals, extractions=[extraction_type],\n",
+    "doc_extractions = create_extractions_dict(session, L_eval, eval_marginals, extractions=[extraction_type],\n",
```

Comment:
Make this line 
`doc_extractions = create_extractions_dict(session, eval_cands, eval_marginals, extractions=[extraction_type]`,\n...`

second argument needs to be eval_cands, not L_eval.

Reply:
Done